### ISSUE TITLE
Addition of dark mode

### ISSUE THREAD

**Issue by mokshmalik5757** [d2l-ai/d2l-en#2603]
Please, add a dark mode for the book. It becomes really difficult to read the book with so much white.

**Comment by yashhalgaonkar** [comment]
+1

**Comment by chabld** [comment]
Agreed. I had to use that extension on Firefox to force dark mode. It' an alright compensation
https://addons.mozilla.org/en-US/firefox/addon/force-dark-mode/?utm_source=addons.mozilla.org

**Comment by yashhalgaonkar** [comment]
> Please, add a dark mode for the book. It becomes really difficult to read the book with so much white.

+1
